contrib/ci/dependencies.xml

```diff
@@ -551,31 +542,33 @@
       <package />
     </distro>
   </dependency>
-  <dependency type="build" id="libappstream-glib-dev">
+<!--
+  <dependency type="build" id="libxmlb-dev">
     <distro id="arch">
-      <package>appstream-glib</package>
+      <package>libxmlb</package>
     </distro>
     <distro id="centos">
-      <package>libappstream-glib-devel</package>
+      <package>libxmlb-devel</package>
     </distro>
     <distro id="fedora">
-      <package>libappstream-glib-devel</package>
+      <package>libxmlb-devel</package>
     </distro>
     <distro id="debian">
       <control>
-        <version>(>= 0.7.4)</version>
+        <version>(>= 0.1.0)</version>
       </control>
       <package variant="x86_64" />
-      <package variant="s390x">libappstream-glib-dev:s390x</package>
+      <package variant="s390x">libxmlb-dev:s390x</package>
       <package variant="i386" />
     </distro>
     <distro id="ubuntu">
       <control>
-        <version>(>= 0.6.9)</version>
+        <version>(>= 0.1.0)</version>
```

Comment:
Since you're using 0.1.1 elsewhere should probably correct this too (even if it's commented out)